pkg/kv/kv.go

```diff
@@ -309,3 +398,15 @@ func (kvs Pairs) ClassifyAndAppend(
 func (kvs Pairs) Clear() Pairs {
 	return kvs[:0]
 }
+
+// NextKey return the smallest []byte that is bigger than current bytes.
+// special case when key is empty, empty bytes means infinity in our context, so directly return itself.
+func NextKey(key []byte) []byte {
```

Comment:
maybe this should also apply https://github.com/pingcap/tidb-lightning/pull/523

Reply:
Done